src/AppBundle/Command/AddSingleUserCommand.php

```diff
@@ -94,6 +96,11 @@ protected function addSingleUser(OutputInterface $output, array $data, array $op
 
         $user->setPassword($this->encodePassword($user, $data['password']));
 
+        if ($data['roleName'] != User::ROLE_ADMIN) {
```

Comment:
@slizzio we discussed adding the client here, as the idea behind the fixture is pre-creating the data

Reply:
Agreed to do this post merge as users are needed to QA registration steps, Ive created a new ticket for this DDPB-1837